guides/source/association_basics.md

```diff
@@ -2091,7 +2092,9 @@ The `collection.size` method returns the number of objects in the collection.
 
 ##### `collection.find(...)`
 
-The `collection.find` method finds objects within the collection. It uses the same syntax and options as `ActiveRecord::Base.find`. It also adds the additional condition that the object must be in the collection.
+The `collection.find` method finds objects within the collection. It uses the same syntax and options as
+[`ActiveRecord::Base.find`](http://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find).
+It also adds the additional condition that the object must be in the collection.
```

Comment:
> The `collection.find` method finds objects within the collection. [...] It also adds the additional condition that the object must be in the collection.

This seems totally redundant to me; I think you were right to remove it initially.

I think what it's _trying_ to say is that `collection.find` behaves identically to `ActiveRecord::Base.find`, except that the query will be scoped to the collection. I think that's clear enough from the first two sentences, though.

WDYT @kaspth?

Reply:
Yes, the line is totally redundant and can be removed. @bogdanvlviv has expressed interest in upping his contributions, so I'm badgering him to explain his understanding and come up with better arguments for it.

In other words: the line shouldn't be removed because "it's wrong" (it ain't), but it should be removed because it's redundant. Bogdan, you know what to do 😊

Reply:
Apologies, I didn't recognise the Socratic method at work here. 😁

Reply:
Done.

Reply:
@eugeneius pretty hidden, so no worries! 😊

@bogdanvlviv cool 👍